agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorTraceExporter.java

```diff
@@ -211,7 +81,7 @@ public CompletableResultCode export(Collection<SpanData> spans) {
     }
 
     CompletableResultCode overallResult = new CompletableResultCode();
-    CompletableResultCode exportResult = exportFn.apply(telemetryItems);
+    CompletableResultCode exportResult = telemetryItemExporter.send(telemetryItems);
```

Comment:
return telemetryItemExporter.send(telemetryItems)?

Reply:
the code here combines the result from telemetryItemExporter.send and the result of any above mapping failures